packages/wix-ui-tpa/src/components/Dialog/Dialog.tsx

```diff
@@ -19,7 +19,9 @@ export interface DialogProps extends TPAComponentProps {
   /** Children to render inside dialog */
   children?: React.ReactNode;
   /** A class name that will be added to the content wrapper */
-  contentClassName?: string;
+  contentWrapperClassName?: string;
+  /** A class name that will be added to the dialog content */
+  dialogContentClassName?: string;
```

Comment:
so let's go with 
```suggestion
 childrenWrapperClassName?: string;
```
I know I know, it's long and not pretty, but because of the original naming we gave to the other prop, I think we need to be very precise with this one.
I hope we'll get to fixing this in future versions.